app/src/main/res/values/colors.xml

```diff
@@ -10,7 +10,7 @@
     <color name="black">#000000</color>
     <color name="white_alpha_70">#b3ffffff</color>
     <color name="black_alpha_70">#b3000000</color>
-    <color name="disabled_button">#9b9b9b</color>
+    <color name="disabled_button">#80008c73</color>
```

Comment:
This changes the behavior in the whole app for quite some different buttons, is this intended

Reply:
according to mocks disabled buttons should be greenish with some transparency. This was intended.

Reply:
mocks = zeplin designs?

Reply:
yes